openpype/plugins/publish/extract_slate_global.py

```diff
@@ -0,0 +1,747 @@
+import os
+import re
+import logging
+import subprocess
+import json
+import platform
+
+import opentimelineio as otio
+from html2image import Html2Image
+
+import pyblish.api
+from openpype.pipeline import publish
+
+class SlateCreator:
+    """
+    Class that formats and renders an html template
+    to use as a slate.
+
+    Html renders as sRGB (or Rec.709 if you want a
+    little gamma boost), so rendering will happen
+    in .png as intermediate file, while then converted
+    to dest cspace using oiio --colorconvert
+    attributes syntax.
+    
+    """
+    
+    def __init__(
+        self,
+        staging_dir="",
+        staging_subfolder="",
+        template_path="",
+        resources_path="",
+        log=None,
+        data={},
+        env={}
```

Comment:
That’s something I did on an internal slate creator to have every parameter for flexibility. It can be skimmed down a lot if need be. I was just too lazy to do it. This probably would need to be a draft pr instead of a normal one 😇